pkg/istio-agent/xds_proxy.go

```diff
@@ -615,6 +622,7 @@ func (p *XdsProxy) buildUpstreamClientDialOpts(sa *Agent) ([]grpc.DialOption, er
 	// Make sure the dial is blocking as we dont want any other operation to resume until the
 	// connection to upstream has been made.
 	dialOptions := []grpc.DialOption{
+		grpc.WithStatsHandler(new(ocgrpc.ClientHandler)),
```

Comment:
need to make it optional, i remember grpc tracing was disabled dual to performance